evtx/Maps/System_Microsoft-Windows-Time-Service_37.map

```diff
@@ -0,0 +1,48 @@
+Author: Hyun Yi @hyuunnn
+Description: Synchronizing system time from time service
+EventId: 37
+Channel: "System"
+Provider: "Microsoft-Windows-Time-Service"
+Maps:
+  -
+    Property: PayloadData1
+    PropertyValue: "ServerAddress: %ServerAddress%"
+    Values:
+      -
+        Name: ServerAddress
+        Value: "/Event/EventData/Data[@Name=\"TimeSource\"]"
+        Refine: "^.*(?=,)"
+  -
+    Property: PayloadData2
+    PropertyValue: "ServerIP: %ServerIP%"
+    Values:
+      -
+        Name: ServerIP
+        Value: "/Event/EventData/Data[@Name=\"TimeSource\"]"
+        Refine: "(?<=-&gt;).*(?=))"
+
+# Documentation:
+# N/A
+#
+# Example Event Data:
+# <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
+#   <System>
+#     <Provider Name="Microsoft-Windows-Time-Service" Guid="{GUID}" /> 
+#     <EventID>37</EventID> 
+#     <Version>0</Version> 
+#     <Level>4</Level> 
+#     <Task>0</Task> 
+#     <Opcode>0</Opcode> 
+#     <Keywords>0x8000000000000000</Keywords> 
+#     <TimeCreated SystemTime="2020-11-07T00:15:17.5759748Z" /> 
+#     <EventRecordID>5549</EventRecordID> 
+#     <Correlation /> 
+#     <Execution ProcessID="16004" ThreadID="2292" /> 
+#     <Channel>System</Channel> 
+#     <Computer>ComputerName</Computer> 
+#     <Security UserID="{UserID}" /> 
+#   </System>
+#   <EventData Name="TMP_EVENT_TIME_SOURCE_REACHABLE">
+#     <Data Name="TimeSource">time.windows.com,0x9 (ntp.m|0x9|0.0.0.0:123->52.231.114.183:123)</Data> 
+#   </EventData>
+# </Event>
```

Comment:
Can you add a new line to the bottom of all of these? Just hit enter at this last line and resubmit.

Reply:
@rathbuna fixed it :)
I was forgotten, sorry